source/develop/release-management/features/storage/single-disk-snapshot.html.md

```diff
@@ -68,41 +71,53 @@ Add <disks> tag to create/restore snapshot.
 
 Note: preview only is not available from rest (restore = preview + commit).
 
-#### Create Snapshot: POST /api/vms/{vm_id}/snapshots
+#### Create Snapshot:
 
-` `<snapshot>
-`   `<vm id="{vm_id}"/>
-`   `<disks>
-`     `<disk id="{disk_id}"/>
-`   `</disks>
-` `</snapshot>
+```xml
+POST /api/vms/{vm_id}/snapshots
 
-#### Restore Snapshot: POST /api/vms/{vm_id}/snapshots/{snapshot_id}/restore
+<snapshot>
+ <vm id="{vm_id}"/>
+ <disks>
+  <disk id="{disk_id}"/>
+ </disks>
+</snapshot>
+```
 
-` `<action>
-`   `<restore_memory>`true|false`</restore_memory>
-`   `<disks>
-`     `<disk id="{disk_id}">
-`       `<image_id>`{image_id}`</image_id>
-`       `<snapshot id="{snapshot_id}"/>
-`     `</disk>
-`   `</disks>
-` `</action>
+#### Restore Snapshot:
+
+```xml
+POST /api/vms/{vm_id}/snapshots/{snapshot_id}/restore
+
+<action>
+ <restore_memory>"true|false"</restore_memory>
+ <disks>
+  <disk id="{disk_id}">
+   <image_id>"{image_id}"</image_id>
+   <snapshot id="{snapshot_id}"/>
+  </disk>
+ </disks>
+</action>
+```
 
 #### oVirt 3.5
```

Comment:
Maybe the "Preview Snapshot" part was added only on 3.5.
In any case it needs rewording.

@danielerez  can you confirm it was added later, in 3.5 ?